Miscellaneous/docker/full_stack_example/docker-compose-windows.yml

```diff
@@ -0,0 +1,174 @@
+---
+version: "2.1"
+services:
```

Comment:
Currently there isn't a huge difference but right now we don't have windows working to confirm. I'm in favour of autogenerating providing it keeps the steps minimal for the user. 

How would you envisage this working? Maybe a template which can be consumed by a script that generates a file for the detected OS?  

Right now we give zip or tar.gz to download from the repo - as otherwise they need to download the whole repo. Open to ideas here as well.